### PR TITLE
Feat/healthcheck version addons

### LINKED ISSUE #104: Add version and healthcheck endpoints
Add a version and healthcheck endpoints to the API.

### PR DESCRIPTION
**Pull request by sylviamclaughlin**
# Summary | Résumé

Created an endpoint to do version return and healthcheck. Closes #104 
Very open to suggestions if there is a better way to check the status of the DynamoDB database. I looked at the available methods and there was nothing that I could see specifically for status checking.

### CODE REVIEW

File: api/tests/routers/test_ops.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+# Test that the healthcheck endpoint returns an error when the table name provided is empty
+@patch.dict(os.environ, {"TABLE_NAME": ""}, clear=True)
+def test_healthcheck_failure_empty_table_name():
+    response = client.get("/healthcheck")
+    assert response.status_code == 200
+    # expected_val = {
+    #     "status": "ERROR",
+    #     "message": "Parameter validation failed:\nInvalid length for parameter TableName, value: 0, valid min length: 3",
+    # }
```
Comment (patheard) [inline api/tests/routers/test_ops.py:56]: Just a minor nit - could you remove this before merging.

**Review by patheard — APPROVED** [review]
Looks good!  Had a tiny non-blocking comment for the unit test.  

I like your idea as well of checking that DynamoDB is responding in the healthcheck endpoint.

File: api/tests/routers/test_ops.py (hunk 1 above)
Comment (sylviamclaughlin) [reply api/tests/routers/test_ops.py]: Removed! Totally did not see it - I guess you stare at it for so long that it becomes invisible. Thanks for noticing it.

Comment (patheard) [reply api/tests/routers/test_ops.py]: :100:

**Review by maxneuvians — APPROVED** [review]
Nice!

**Pull request merged**